app/src/main/res/values/strings.xml

```diff
@@ -1,81 +1,179 @@
 <resources>
+    <!-- LOCALIZATION NOTE (app_name): The string 'app_name' will be used any time the application
+         name is to be displayed to the user inside of the application as well as any system launcher
+         such as the text bellow the application launch icon. -->
     <string name="app_name">Firefox Reality</string>
-    <string name="new_tab">New tab</string>
-    <string name="menu_enter_private_browsing">Enter Private Browsing</string>
-    <string name="menu_exit_private_browsing">Exit Private Browsing</string>
-    <string name="menu_focus_mode">Focus Mode</string>
+    <!-- LOCALIZATION NOTE (keyboard_enter_label): The string 'keyboard_enter_label' is used on the
```

Comment:
same comment as above: since the localization note will appear directly above the string, you can just say "this string" (some localizers tend to not look at string ids anyways, so this may be confusing for them)